services/github/github-downloads.service.js

```diff
@@ -247,7 +247,7 @@ export default class GithubDownloads extends GithubAuthV3Service {
     }
 
     if (releases.length === 0) {
-      throw new NotFound({ prettyMessage: 'no releases' })
+      throw new NotFound({ prettyMessage: 'no releases found' })
```

Comment:
The issue here is that `fetchLatestRelease()` (which we also call in this service) throws "no releases found". I've brought the message for both services into line.